perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/SecurityTeamsManagerMethod.java

```diff
@@ -120,10 +121,50 @@ public SecurityTeam call(ApiCaller ac, Deserializer parms) throws PerunException
 	getAdmins {
 		@Override
 		public List<User> call(ApiCaller ac, Deserializer parms) throws PerunException {
```

Comment:
There is javadoc just for method without "onlyDirectAdmins" parameter. Please add this new one.